config/routes.rb

```diff
@@ -391,15 +390,25 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
-      get '/by_mail' => 'gpo_verify#index', as: :gpo_verify
-      post '/by_mail' => 'gpo_verify#create'
+      get '/by_mail' => 'by_mail/enter_code#index', as: :verify_by_mail_enter_code
+      post '/by_mail' => 'by_mail/enter_code#create'
       get '/by_mail/confirm_start_over' => 'confirm_start_over#index',
           as: :confirm_start_over
 
       if FeatureManagement.gpo_verification_enabled?
-        get '/usps' => 'gpo#index', as: :gpo
-        put '/usps' => 'gpo#create'
+        get '/usps' => 'by_mail/request_letter#index', as: :request_letter
+        put '/usps' => 'by_mail/request_letter#create'
       end
+
+      get '/come_back_later' => 'by_mail/letter_enqueued#show', as: :letter_enqueued
+
+      # BEGIN temporary routes in preparation for renaming the GPO routes
+      #       These will allow old instances to serve requests for new routes during the 50/50
+      #       state when new routes are deployed.
+      get '/by_mail/request_letter' => 'by_mail/request_letter#index'
+      get '/by_mail/letter_enqueued' => 'by_mail/letter_enqueued#show'
+      get '/by_mail/enter_code' => 'by_mail/enter_code#index'
```

Comment:
I think you need the put routes here too.

Reply:
The actual forms all set their `action` attributes at the old path names (for example, [this one](https://github.com/18F/identity-idp/blob/5d2f930f97e0bcd10fff8c88a8934380aa83992e/app/views/idv/by_mail/enter_code/index.html.erb#L63)) , so no PUT routes are needed here. We WILL need to do a second set of PUT routes when we ultimately rename the routes, but that'll be a different PR